example_test.go

```diff
@@ -27,6 +28,10 @@ func thereShouldBeEvidenceOfASuccessfulHttpRequest() error {
 	if resp.StatusCode != 200 {
 		return fmt.Errorf("Bad status code: %v", resp.StatusCode)
 	}
+	var str = "https://postman-echo.com/get?foo1=foo+&foo2=bar"
+	if !strings.Contains(resp.Body, str) {
+		return fmt.Errorf("Response body: %v doesn't contain sting: %v", resp.Body, str)
```

Comment:
Why don't we use `t *testing.T` instead of `fmt` here

Reply:
@jmromanos i tried but when test fails, logging-wise ends up screwing with cucumber (godog)'s log.

Reply:
Example:
```
runtime error: invalid memory address or nil pointer dereference
runtime.call64
	/usr/local/Cellar/go/1.11.5/libexec/src/runtime/asm_amd64.s:523
runtime.gopanic
	/usr/local/Cellar/go/1.11.5/libexec/src/runtime/panic.go:513
runtime.panicmem
	/usr/local/Cellar/go/1.11.5/libexec/src/runtime/panic.go:82
runtime.sigpanic
	/usr/local/Cellar/go/1.11.5/libexec/src/runtime/signal_unix.go:390
_/Users/darocfil/personal/tranquility-example-cucumber.thereShouldBeEvidenceOfASuccessfulHttpRequest
	/Users/darocfil/personal/tranquility-example-cucumber/example_test.go:34
runtime.call32
```

instead of:
```
--- Failed steps:

  Scenario: Get of postmanecho.com # features/example/basic.feature:7
    Then there should be evidence of a successful http request # features/example/basic.feature:10
      Error: Response body: {"args":{"foo1":"foo ","foo2":"bar"},"headers":{"x-forwarded-proto":"https","host":"postman-echo.com","content-length":"47","accept-encoding":"gzip","user-agent":"Go-http-client/1.1","x-forwarded-port":"443"},"url":"https://postman-echo.com/get?foo1=foo+&foo2=bar"} doesn't contain sting: https://postman-echo.com/get?foo1aa=foo+&foo2=bar
```